tools/build_pip_package.py

```diff
@@ -0,0 +1,57 @@
+# Copyright 2023 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""This module is used for building the Gematria python package. Usage:
+
+bazel run //tools:build_pip_package bdist_wheel
```

Comment:
We probably need to do `bazel run -c opt //tools:build_pip_package -- bdist_wheel`

1. `-c opt` so that the C++ code is built with optimization on (which makes a big difference).
2. we need the `--` to separate bazel flags from setuptools flags.

Reply:
I definitely did not realize that Bazel didn't run optimizations by default. Thanks for pointing this out!

Reply:
The default is [`fastbuild`](https://bazel.build/docs/user-manual#build-semantics), which doesn't optimize and does not define `-DNDEBUG` so it also runs all assertions and extra checks.